packages/lumx-core/src/css/design-tokens.css

```diff
@@ -298,11 +298,11 @@
     --lumx-tabs-link-emphasis-selected-state-active-theme-light-color: var(--lumx-color-dark-L1);
     --lumx-tabs-link-emphasis-selected-state-active-theme-light-border-color: var(--lumx-color-primary-N);
     --lumx-tabs-link-emphasis-selected-state-active-theme-dark-background-color: var(--lumx-color-light-L4);
-    --lumx-tabs-link-emphasis-selected-state-active-theme-dark-color: var(--lumx-color-light-N);
+    --lumx-tabs-link-emphasis-selected-state-active-theme-dark-color: var(--lumx-color-light-L1);
     --lumx-tabs-link-emphasis-selected-state-active-theme-dark-border-color: var(--lumx-color-light-N);
     --lumx-text-field-input-min-height: 36px;
-    --lumx-text-field-input-padding-horizontal: 12px;
-    --lumx-text-field-input-border-radius: 4px;
+    --lumx-text-field-input-padding-horizontal: var(--lumx-spacing-unit-medium);
```

Comment:
spacing medium is not documented, should we ? or is it a barely used value that we do not what to propagate ?

Reply:
barely used but we could document it